phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataProtocol.java

```diff
@@ -99,7 +99,7 @@ public abstract class MetaDataProtocol extends MetaDataService {
     public static final long MIN_SYSTEM_TABLE_TIMESTAMP_4_15_0 = MIN_TABLE_TIMESTAMP + 29;
     public static final long MIN_SYSTEM_TABLE_TIMESTAMP_4_16_0 = MIN_TABLE_TIMESTAMP + 33;
     public static final long MIN_SYSTEM_TABLE_TIMESTAMP_5_1_0 = MIN_SYSTEM_TABLE_TIMESTAMP_4_16_0;
-    public static final long MIN_SYSTEM_TABLE_TIMESTAMP_5_2_0 = MIN_TABLE_TIMESTAMP + 37;
+    public static final long MIN_SYSTEM_TABLE_TIMESTAMP_5_2_0 = MIN_TABLE_TIMESTAMP + 39;
```

Comment:
Why the increase of 2 (from 37 to 39)?

Reply:
To increase timestamp by two for 5.2 to add TTL and ROWKEY_PREFIX columns here :- https://github.com/apache/phoenix/blob/40a30ac66a7655003f52e816525c3e209d44a083/phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java#L4135